tests/unit/h/security/policy/_cookie_test.py

```diff
@@ -96,3 +118,7 @@ def test_methods_add_vary_callback(
     @pytest.fixture
     def user(self, factories):
         return factories.User()
+
+    @pytest.fixture(autouse=True)
+    def is_api_request(self, mocker):
+        return mocker.patch("h.security.policy._cookie.is_api_request")
```

Comment:
Using [pytest-mock](https://pytest-mock.readthedocs.io/en/latest/) (the `mocker` fixture) nowadays instead of [Hypothesis's ancient, custom `patch` fixture](https://www.seanh.cc/2017/03/17/patch/).